### PR TITLE
Saas dev publishing episode view

### PR DESCRIPTION
**Pull request by dpsommer**
Left-hand pane view for podcast episodes for more fluid scheduling

### CODE REVIEW

File: airtime_mvc/application/controllers/LibraryController.php [hunk 1]
```diff
@@ -222,6 +222,7 @@ public function deleteAction()
         $message = null;
         $noPermissionMsg = _("You don't have permission to delete selected items.");
 
+        Logging::info($mediaItems);
```
Comment (asantoni) [inline airtime_mvc/application/controllers/LibraryController.php]: Nix the logging.

File: airtime_mvc/application/common/PodcastManager.php [hunk 2]
```diff
@@ -64,7 +64,7 @@ protected static function _findUningestedEpisodes($podcast, $service) {
             //  episodes in the list of episodes to ingest, don't skip this episode - we should try to ingest the
             //  most recent episode when the user first sets the podcast to automatic ingest.
             // If the publication date of this episode is before the ingest timestamp, we don't need to ingest it
-            if ((empty($ts) && !empty($episodes)) || strtotime($episodeData["pub_date"]) < strtotime($ts)) {
```
Comment (asantoni) [inline airtime_mvc/application/common/PodcastManager.php:67]: What timezone is the publication date in? Is that the same timezone the auto-ingest timestamp is in? (Are they always the same?)

What if the timezone is missing from the publication date string?

File: airtime_mvc/public/js/airtime/library/library.js [hunk 3]
```diff
@@ -1230,38 +1249,104 @@ var AIRTIME = (function(AIRTIME) {
...
+        return oTable;
+    };
+
+    mod.openPodcastEpisodeDialog = function () {
+        var episode = mod.podcastEpisodeTableWidget.getSelectedRows()[0];
+        $("body").append("<div id='podcast_episode_dialog'></div>");
+        var dialog = $("#podcast_episode_dialog").html(episode.description);
+        dialog.html(dialog.text());
+        dialog.dialog({
+            title: $.i18n._(episode.title),
```
Comment (asantoni) [inline airtime_mvc/public/js/airtime/library/library.js]: This one shouldn't be wrapped in an $.i18n_.() function. It's user data, right?

File: airtime_mvc/public/js/airtime/library/podcast.js [hunk 4]
```diff
@@ -436,51 +536,114 @@ var AIRTIME = (function (AIRTIME) {
...
+                    var self = this;
+                    $(nRow).draggable({
+                        helper: function () {
+                            var $row = $(this), data = self._datatable.fnGetData(nRow);
+                            $row.data("aData", data.file);
+                            self.selectRow(this, data, self.SELECTION_MODE.SINGLE, $row.index());
+                            var selected = self.getSelectedRows().length, container,
+                                width = self._$wrapperDOMNode.closest(".dataTables_wrapper").outerWidth(), message;
+
+                            message = sprintf($.i18n._("Adding %s Item%s"), selected, selected > 1 ? "s" : "");
```
Comment (asantoni) [inline airtime_mvc/public/js/airtime/library/podcast.js]: This way of doing pluralization doesn't work for some languages. Pluralization is built into the jQuery localization plugin we're using. Search this page for "plural" to see the right way to do this:
https://github.com/wikimedia/jquery.i18n

Comment (asantoni) [inline airtime_mvc/public/js/airtime/library/podcast.js]: Also make sure to add any strings you want localized in JavaScript to generalTranslationTableAction() in LocaleController.php.

File: airtime_mvc/public/css/styles.css [hunk 5]
```diff
@@ -4029,9 +4037,33 @@ li .ui-state-hover {
...
+
 /* UI Revamp Video */
 
 #whatsnew {
+    display: -webkit-box;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: -webkit-flex;
     display: flex;
     flex-direction: column;
```
Comment (asantoni) [inline airtime_mvc/public/css/styles.css:4073]: Doesn't flex-direction have to be prefixed too?

File: airtime_mvc/application/common/PodcastManager.php (hunk 2 above)
Comment (dpsommer) [inline airtime_mvc/application/common/PodcastManager.php:67]: Publication dates have to comply with [RFC-822](https://www.ietf.org/rfc/rfc0822.txt), so they have to include a timestamp (see also [PHP's RFC-822 format string](http://php.net/manual/en/class.datetime.php#datetime.constants.rfc822)). strtotime converts the timestamps to Unix format, so the comparison should be timezone agnostic (to be consistent, the timestamps are stored after being passed through gmdate).

**Comment by asantoni** [comment]
Looks good once those couple little things are fixed. Merge it whenever you're done.

**Pull request merged**